### PR TITLE
Autopay fail email

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?190226

@benrudolph cc: @dannyroberts

### CODE REVIEW

File: corehq/apps/accounting/tasks.py [hunk 1]
```diff
@@ -351,6 +355,41 @@ def send_purchase_receipt(payment_record, core_product, domain,
     )
 
 
+@task(ignore_result=True)
```
Comment (benrudolph) [inline corehq/apps/accounting/tasks.py]: may want to put this on the `queue=background_queue`. probably doesn't make sense to eat resources from tasks where users are actually waiting in the browser

**Comment by benrudolph** [comment]
legit, will let you respond to comment

**Comment by dannyroberts** [comment]
(not mergeable)

**Comment by nickpell** [comment]
updated

File: corehq/apps/accounting/tasks.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/accounting/tasks.py]: makes sense, done.  Might make sense to do this for more of the billing tasks as well.

**Comment by benrudolph** [comment]
nice hopefully this'll get us a little more :dollar: bills. +1 to making other billing tasks background

**Pull request merged**